src/types/card.interface.ts

```diff
@@ -0,0 +1,63 @@
+interface card {
+  id: number;
+  list: Array<cardData>;
+}
+
+interface cardData {
+  id: string | number;
+  cardCVC: string;
+  cardNickname: string;
+  cardNumbers: cardNumbers;
+  cardPassword: string;
+  cardType: cardType;
+  expireDate: expireDate;
+  ownerName: ownerName;
+}
+
+interface cardType {
+  name: string;
+  color: string;
+}
+
+interface cardNumbers {
+  firstNumber: string;
+  secondNumber: string;
+  thirdNumber: string;
+  fourthNumber: string;
+}
```

Comment:
이거 타입핑이 반복되는데 Record 어때요..? (나도 잘 모름, 잘 안써봤음. 그냥 Record 간지나고 더 안전하대서 제안함)

```suggestion

type CardNumbers = Record<'firstNumber' | 'secondNumber' | 'thirdNumber' | 'fourthNumber', number>

```

Reply:
오... 레코드.. 처음봅니다... 👍👍

추가로 찾아보니 Record가 나오기 전까지는 저런 방식으로 했었다고 하네 😎
index signature 이라고 치니까 나옵니다!

Reply:
👏 👏 👏